.github/workflows/compressed.yml

```diff
@@ -0,0 +1,27 @@
+name: Compressed Size
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
You need to add the `canary` branch here, because currently all PRs are to canary :)

Reply:
Fair enough!